cinema/models.py

```diff
@@ -35,8 +43,8 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre, related_name="movies")
+    actors = models.ManyToManyField(Actor, related_name="movies")
```

Comment:
The `related_name` 'movies' is used for both the `genres` and `actors` fields in the `Movie` model. This can cause conflicts when accessing related objects. Consider using unique `related_name` values for each field to avoid these conflicts.